german/schwimmen/hunger/hunger.md

```diff
@@ -2,3 +2,5 @@ Alle schauen sich um. Wo bekommt man jetzt möglichst schnell was zu essen her?
 Im Schwimmbad hat man nur eine Möglichkeit. Am Kiosk hinten rechts.
 Also hin dort!
 
+[Moment mal - Kiosk?](momentmal/momentmal.md)
+[Renn so schnell wie möglich, damit du der erste bist.](rennen/rennen.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.